src/external/atomsbase.jl

```diff
@@ -19,15 +19,23 @@ function parse_system(system::AbstractSystem{D}) where {D}
     atoms = map(system) do atom
         pseudo = get(atom, :pseudopotential, "")
         if isempty(pseudo)
-            ElementCoulomb(atomic_number(atom))
+            ElementCoulomb(atomic_number(atom); mass=atomic_mass(atom))
         else
-            get!(cached_pspelements, pseudo) do
+            key = pseudo * string(get(atom, :atomic_mass, ""))
```

Comment:
Unlike `:pseudopotential`, which is an optional AtomsBase property, all atoms are expected to have a mass, so you can just use `atomic_mass(atom)`.